src/Command/Helpers/DrushWrapper.php

```diff
@@ -98,4 +104,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     return $exit_code;
   }
 
+  /**
+   * Returns the output in the specified format.
+   *
+   * @param string $output
+   *   The output to format.
+   * @param bool $json
+   *   If true, formats the output into a json string.
+   *
+   * @return string
+   *   The formatted output.
+   */
+  protected function formatOutput(string $output, bool $json): string {
```

Comment:
I think we can update return type hint as `mixed` as json_decode() is expected to return mixed type.

Reply:
Good point!